admin/views/tabs/dashboard/integrations.php

```diff
@@ -1,50 +0,0 @@
-<?php
-/**
- * WPSEO plugin file.
- *
- * @package WPSEO\Admin\Views
- *
- * @uses Yoast_Form $yform Form object.
- */
-
-use Yoast\WP\SEO\Presenters\Admin\Alert_Presenter;
-
-if ( ! defined( 'WPSEO_VERSION' ) ) {
-	header( 'Status: 403 Forbidden' );
-	header( 'HTTP/1.1 403 Forbidden' );
-	exit();
-}
-
-$integration_toggles = Yoast_Integration_Toggles::instance()->get_all();
```

Comment:
I think we can deprecate `Yoast_Integration_Toggles` too: it seems to be used just here.

Reply:
Those are still used in the network options